crates/sui-framework/sources/event.move

```diff
@@ -2,6 +2,22 @@
 // SPDX-License-Identifier: Apache-2.0
 
 module sui::event {
+    /// An attachable event. Should be emitted once unpacked from a
+    /// a wrapper. Holds any custom event data inside.
+    struct Event<T: copy + drop + store> has store {
```

Comment:
I think this is probably a bit misleading without some other identifier, since you can emit non-`Event` events. Even typing that is confusing...